spec/models/course_spec.rb

```diff
@@ -8,14 +8,14 @@
     it "is valid" do
       expect(subject).to be_valid
     end
-
   end
 
   context "with invalid attributes" do
     let(:course) { build(:course, name: nil) }
 
     it "is valid" do
-      expect(subject).to be_valid
+      expect(subject).to_not be_nil
```

Comment:
Is it worth also checking that the created course has a name?
It's not quite clear whether this is testing that your course can be created without a name, or if the factory assigns a default name.